app/helpers/file_attachment_helper.rb

```diff
@@ -14,6 +14,7 @@ def file_attachment_attributes(attachment_revision, document)
       file_size: attachment_revision.byte_size,
       number_of_pages: attachment_revision.number_of_pages,
       url: preview_file_attachment_path(document, attachment_revision.file_attachment),
+      unique_reference: attachment_revision.unique_reference,
```

Comment:
I believe we'll only want to do this for documents with featured attachments as this is a field only allowed in publication assets: https://github.com/alphagov/govuk-content-schemas/blob/ca3f442da137444b246111fb4b3579835cbd2ff7/formats/shared/definitions/asset_links.jsonnet#L32-L40

Reply:
Updated.